### PR TITLE
[OAP-1610][SQL-Data-Source-Cache][SDLe]Modify pom.xml to pass SDLe BDBA binaries vulnerabilities scanning

### PR DESCRIPTION
**Pull request by HongW2019**
## What changes were proposed in this pull request?
OAP will release as a binary product, and offer users jars, so it should comply with intel SDL rules. After executing BDBA scanning binaries for known vulnerabilities, some jar dependencies should exclude or upgrade 3rd party  components which have security vulnerabilities in the pom.xml.

Fixed: #1610    [SQL Data Source Cache]
## How was this patch tested?

Pass BDBA scanning for known vulnerabilities.

Also pass vmemcache Binary 3TB TPCDS

### CODE REVIEW

File: oap-cache/oap/pom.xml [hunk 1]
```diff
@@ -419,6 +419,14 @@
               <include>org.reflections:reflections</include>
             </includes>
           </artifactSet>
+          <filters>
+            <filter>
+              <artifact>org.apache.parquet:*</artifact>
+              <excludes>
+                <exclude>shaded/**</exclude>
```
Comment (HongW2019) [inline oap-cache/oap/pom.xml:426]: shaded/**( including Jackson and libthrift0.9.3) are excluded from dependency Parquet1.10.1 to be maven into oap-cache-0.9.0-with-spark-3.0.0.jar, will this exclusion from oap-cache jar affect yours? if not, we will exclude them to pass BDBA vulnerabilties scanning @LuciferYang

Comment (LuciferYang) [reply oap-cache/oap/pom.xml:426]: should be no negative impact

Comment (HongW2019) [reply oap-cache/oap/pom.xml:426]: Okay, Thanks.

**Comment by zhixingheyi-tian** [comment]
@HongW2019  Please make sure that there are no problems in the integration testing.

**Pull request merged**